CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Updated cloudwatch log stream ID to have attendee_id
+- Ability to choose default in chrome browsers 
```

Comment:
This is a bit misleading. We support choosing default in Chrome before so could you add more details as well as the PR title?

Reply:
Selecting the 'default' in chrome never actually chose the 'default' in Chrome. Please let me know what should we have here instead

Reply:
Not only Chrome, but also Opera also lists the same device twice

Reply:
> Selecting the 'default' in chrome never actually chose the 'default' in Chrome. Please let me know what should we have here instead

Before this PR, choosing `default` works if you have not selected any device or have seleted a non-`default` device before.